python/profiler.cpp

```diff
@@ -4,6 +4,7 @@
 #include <pybind11/stl.h>
 
 #include <arbor/profile/meter_manager.hpp>
+#include <arbor/profile/profiler.hpp>
```

Comment:
You also need to add `#include <arbor/version.hpp>` to pick up `ARB_PROFILE_ENABLED`